doc/source/user_guide/computation.rst

```diff
@@ -571,6 +571,29 @@ and we want to use an expanding window where ``use_expanding`` is ``True`` other
    3     3.0
    4    10.0
 
+.. versionadded:: 1.1
+
+For some problems knowledge of the future is available for analysis. For example, this occurs when
+each data point is a full time series read from an experiment, and the task is to extract underlying
+conditions. In these cases it can be useful to perform forward-looking rolling window computations.
+``FixedForwardWindowIndexer`` class is available for this purpose. This ``BaseIndexer`` subclass
+implements a closed fixed-width forward-looking rolling window, and we can use it as follows:
+
+.. code-block:: ipython
```

Comment:
use an ipython block here

Reply:
@jreback 
Sorry, could you clarify? To me, my ipython block looks right, both in raw and in the docs I compiled locally. Do I need to do something differently?

Reply:
we don't use code-blocks on executable code. pls use an ipython block so that the doc-build executes this, that way it will never get out of date.